engine/spec.go

```diff
@@ -97,9 +98,10 @@ type (
 	// VolumeHostPath mounts a file or directory from the
 	// host node's filesystem into your container.
 	VolumeHostPath struct {
-		ID   string `json:"id,omitempty"`
-		Name string `json:"name,omitempty"`
-		Path string `json:"path,omitempty"`
+		ID       string `json:"id,omitempty"`
+		Name     string `json:"name,omitempty"`
+		Path     string `json:"path,omitempty"`
+		ReadOnly bool   `json:"read_only,omitempty"`
```

Comment:
The `ReadOnly` attribute can be removed since there is no `ReadOnly` flag associated with a [HostPathVolumeSource](https://godoc.org/k8s.io/api/core/v1#HostPathVolumeSource) in Kubernetes.